test/WebJobs.Script.Tests.Integration/NodeEndToEndTests.cs

```diff
@@ -1229,6 +1229,12 @@ public void ExcludedFunction_NotAddedToHost()
             Assert.Equal(1, array[1]);
         }
 
+        public void Dispose()
+        {
+            // Make sure all logs are flushed before next test.
```

Comment:
if only it was that easy ;)

This will get the host, but all of your function tracewriters & ILoggers are off on their own